test/functional/legacy/083_tag_search_with_file_encoding_spec.lua

```diff
@@ -0,0 +1,110 @@
+-- Tests for tag search with !_TAG_FILE_ENCODING.
+
+local helpers = require('test.functional.helpers')
+local insert, source, clear, expect, write_file = helpers.insert,
+  helpers.source, helpers.clear, helpers.expect, helpers.write_file
+
+local has_iconv = function()
+  return os.execute(helpers.nvim_prog .. ' -u NONE -i NONE --embed --cmd ' ..
+    [['if has("iconv")|q|else|cq|endif']]) == 0
+end
```

Comment:
I think you mean `has("iconv")`, I could not find the string "iconv2" anywhere in the source.

The old comment was removed in the commit that fixed the issue. The problem was that I did not pass an empty function to `pending` on the second arg.